src/components/Login.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import "./login.css";
+import {NavLink} from "react-router-dom";
+
+function Login(props) {
+
+    return (
+        <>
+            <h1>Login</h1>
+
+            <div id="login">
+
+                <form>
+                    <fieldset>
+                        <label htmlFor="userName">Username</label>
+                        <div className="field" id="fields">
+                            <input
+                                id="userName"
```

Comment:
this id is unnecessary. if you need to select this element, use `fieldset:first-of-type > div > input`

Reply:
or actually I see you're using it for `htmlFor`, so you can keep it. but then please change the id to `username`